packages/lit-dev-content/site/learn/learn.11tydata.js

```diff
@@ -0,0 +1,190 @@
+  data.date = new Date(data.date);
+  delete data.description;
+  delete data.header;
+  delete data.steps;
+  return data;
+};
+
+const loadArticleData = async () => {
+  const articlesDir = path.resolve(__dirname, '../articles/article');
+  const files = await fs.readdir(articlesDir);
+  const markdownArticles = files.filter((filename) => filename.endsWith('.md'));
+
+  const articleData = Promise.all(
+    markdownArticles.map(async (articleFileName) => {
+      const contents = await fs.readFile(
+        path.resolve(articlesDir, articleFileName),
+        {
+          encoding: 'utf8',
+        }
+      );
+      const frontMatter = matter(contents);
+      return {
+        kind: 'article',
+        url: `/articles/${articleFileName.slice(0, -3)}/`,
+        date: new Date(frontMatter.publishDate),
+        ...frontMatter.data,
+      };
+    })
+  );
+  return articleData;
+};
+
+// Note: Img can be obtained with: https://i.ytimg.com/vi_webp/<video id>/mqdefault.webp
+// or https://i.ytimg.com/vi/<video id>/mqdefault.jpg
+const loadVideoData = () =>
+  [
+    {
+      title: 'Chat with Lit #1 – Westbrook Johnson (Adobe) ',
+      summary: `Listen in on this live-recorded Twitter Space episode, hosted by Rody Davis (@rodydavis) and Elliott Marquez (@techytacos), with guest Westbrook Johnson (@WestbrookJ) from Adobe.`,
+      youtubeId: 'it-NXhxkOJo',
+      date: "Jul 23 2021",
+    },
+    {
+      title: 'How to build your first Lit component',
+      summary: `Learn how to build your first Lit component and use it with React, Vue, and in a markdown editor.`,
+      youtubeId: 'QBa1_QQnRcs',
+      date: "Apr 25 2022",
+    },
+    {
+      title: 'What are elements - Lit University (Basics)',
+      summary: `Software Engineer Elliott Marquez shares what elements are, how to make, and interact with them. Learn about the basic building block of the web in this video!`,
+      youtubeId: 'x_mixcGEia4',
+      date: "Apr 27 2022",
+    },
+    {
+      title: 'How to build a carousel in Lit',
+      summary: `In this video, we build a simple-carousel using Lit, letting us explore passing children into your web component, and web component composition.`,
+      youtubeId: '2RftvylEtrE',
+      date: "May 3 2022",
+    },
+    {
+      title:
+        'Event communication between web components - Lit University (Advanced)',
+      summary: `Follow along as Lit Software Engineer Elliott Marquez shares the pros, cons, and use cases of communicating with events.`,
+      youtubeId: 'T9mxtnoy9Qw',
+      date: "May 5 2022",
+    },
+    {
+      title: 'How to style your Lit elements',
+      summary: `We cover how the Shadow DOM works, illustrate the benefits of encapsulated CSS, and show you how to use CSS inheritance, custom properties and shadow parts to expose a flexible public styling API.`,
+      youtubeId: 'Xt7blcyuw5s',
+      date: "Oct 3 2022",
+    },
+    {
+      title: 'Introduction to Lit - Lit University (Basics)',
```

Comment:
I think the hyphen and the parentheses make this title a bit noisy.

The _Build With Lit_ series doesn't have the series names in the title, do the _Lit University_ videos need that?

Reply:
Agree about noise. I've removed the series from the title. (It's visible in the thumbnail)